src/scr_fetch.c

```diff
@@ -774,14 +798,17 @@ static int scr_fetch_summary(
     /* the key is the source rank, which we don't care about,
      * the info we need is in the element hash */
     scr_hash* elem_hash = scr_hash_elem_hash(elem);
-
+    scr_dbg(1, "scr_fetch_summary print at %d:%s", __LINE__, __FILE__);
+    scr_hash_log(elem_hash, 1, 0);
     /* get pointer to file hash */
     scr_hash* file_hash = scr_hash_get(elem_hash, SCR_SUMMARY_6_KEY_FILE);
     if (file_hash != NULL) {
       /* TODO: parse summary file format */
+      scr_dbg(1, "scr_fetch_summary at %d:%s", __LINE__, __FILE__);
       scr_hash_merge(file_list, elem_hash);
     } else {
-      rc = SCR_FAILURE;
```

Comment:
Ah, I see your point.  Yes, that's a good solution.  Our current restart interface is somewhat buggy in the case when a process has no files.  We'll fix the API to help with this case in the next revision.